parsec/core/fs/workspacefs/sync_transactions.py

```diff
@@ -415,8 +415,18 @@ async def file_conflict(
                 await self.local_storage.set_manifest(parent_id, new_parent_manifest)
                 await self.local_storage.set_manifest(entry_id, other_manifest)
 
-                self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=new_manifest.id)
-                self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent_id)
+                self._send_event(
+                    CoreEvent.FS_ENTRY_UPDATED,
+                    id=parent_id,
+                    reason=FSEntryUpdatedReason.SYNC_FILE_CONFLICT,
+                )
```

Comment:
I would add `entry_id`, `entry_name`, `entry_conflict_id` and `entry_conflict_name` to this event

Reply:
I am not sure about the `entry_conflict_id`, is it not the same as `parent_id`?